File: src/main/java/stair_case/StairCase.java
package stair_case;

public class StairCase {
    public void stairCase(){
        for(int star = 1; star<=6; star++){
            int space = 6-star;
            for(int i = 0; i<space; i++){
                System.out.print(' ');
            }

            for(int i=0; i<star; i++){
                System.out.print('*');
            }
            System.out.println();
        }
    }
}

File: src/main/kotlin/stair_case/stairCase.kt
package stair_case

fun stairCase() {
    for (star in 1..6) {
        val space = 6 - star
        for (i in 0 until space){
            print(' ')
        }
        for (i in 0 until star){
            print('#')
        }
        println()
    }
}